salsa_dancing_molecules/simulations/nve.py

```diff
@@ -35,16 +37,26 @@ def run_for_materials(formula, api_key, steps, output_path, repeat=0):
         if repeat > 0:
             atoms = atoms.repeat(repeat)
 
-        try:
-            print(f'Simulating for {symbols}')
-            run(atoms, steps, output_name)
-        except AsapError as e:
-            print(f'ASAP3 error for {symbols}: {e}')
-            print('Trying without ASAP3...')
-            run(atoms, steps, output_name, False)
-
-
-def run(atoms, steps, output_path, use_asap=True):
+        while True:
```

Comment:
I don't think there is any problem with this loop. However we dont want the loop to be stuck in infinite loop, I don't see any issues for this loop but I could be wrong. I just want to request if you can doublecheck if they're might be any chance for the existence of infinite loop? If they're is no chance, then just comment it here.